utils/build/webpack.umd.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+const path = require('path');
+const merge = require('webpack-merge');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const baseConfig = require('./webpack.base');
+const packageManifest = require(path.resolve('package.json'));
+
+module.exports = merge(baseConfig, {
+  output: {
+    path: path.resolve('dist', 'umd'),
+    filename: 'bundle.min.js',
+    library: packageManifest['zendeskgarden:library'],
+    libraryTarget: 'umd'
+  },
+  optimization: {
+    minimizer: [
+      new UglifyJsPlugin({
+        cache: true,
+        parallel: true,
+        sourceMap: true
+      })
+    ]
+  },
+  externals: {
```

Comment:
It looks like some of the external names might be messed up. You might want to double check them.